apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -63,6 +67,12 @@ export const styles = {
     lineHeight: '52px',
     width: 210
   },
+  col5: {
```

Comment:
That seems fine, but in that case it seems as though the name could describe the layout in some way other than  just location.